.github/workflows/quality-checks.yml

```diff
@@ -28,10 +30,17 @@ jobs:
           poetry install
 
       - name: Run pytest
-        run: poetry run pytest
+        run: poetry run pytest --cov --cov-report=xml --junitxml=junit.xml -o junit_family=legacy
```

Comment:
Is there a reason to go with XML?

Reply:
It's to enable https://app.codecov.io/github/uktrade/platform-tools/tests/dbtp-1511-add-codecov, which I think also does the comments in the pull requests.